WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -372,6 +372,10 @@ import WordPressComAnalytics
         resultsStatusView = WPNoResultsView()
     }
 
+    private func setupNavigationItem() {
+        // Hide the back button title
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: " ", style: .Plain, target: nil, action: nil)
```

Comment:
It would be nice if we could set this via the story board instead of in code. I think just adding a space to the `Reader` UINavigationItem's `Back Button` field in the storybaord would do the trick.